docs/release_notes.adoc

```diff
@@ -39,6 +39,14 @@ subscriber.receive(_) >> _
 - `IterationInfo` now has a property for the `iterationIndex` and one for a map of data variable names to values.
   This is typically not interesting for the average user, but might be helpful for authors of Spock extensions.
 
+- Data driven features are now unrolled by default. `@Unroll` can still be used to specify a custom naming pattern.
+
+- `@Rollup` can now be used on feature and spec level to explicitly roll up any feature where the reporting of single
+  iterations is not wanted.
+
+- The system property `spock.globalRollup` can be set to `true` to roll up all features by default if not overwritten
```

Comment:
+1 for having a flag to make the project globally behave as Spock 1.3 with unrolling (although I hope it will not be used too often).